<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HVlcPlayer</class>
 <widget class="QMainWindow" name="HVlcPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>播 放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>暂 停</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>停 止</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>媒体(&amp;M)</string>
    </property>
    <addaction name="action_101"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>播放(&amp;P)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>音频(&amp;A)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>视频(&amp;V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>字幕(&amp;S)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>视图(&amp;I)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
   </widget>
   <addaction name="menu_1"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
   <addaction name="menu_8"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_101">
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
